week-03/main.js

```diff
@@ -13,9 +20,16 @@ var list = [1, 2, 3, 4, 5];
 // and `hobbies` an array.
 // Assign the object to the variable `user`
 
+var user = {
+  name : "alex",
+  isAlive : true,
+  hobbies : ["drawing", "smoking", "drinking", "hiking", "pets"]
+};
+
 
 
 // Add 'golf' to the `user`'s `hobby` array.
+user.hobbies = "golf"l
```

Comment:
This reassigns the `hobbies` array to a string. Then there's a syntax error after. How would you add `golf` to `hobbies`?